vue/src/vueTextMask.js

```diff
@@ -1,23 +1,91 @@
 import createTextMaskInputElement from '../../core/src/createTextMaskInputElement'
 
 export default {
-  textMaskInputElement: null,
+  template: `
+    <input ref="input"
+      :value="value"
+      @input="updateValue($event.target.value)"
+    />
+  `,
 
-  inputHandler({target: {value}}) {
-    return this.textMaskInputElement.update(value)
+  name: 'masked-input',
+
+  props: {
+    value: {
+      type: String,
+      required: false,
+      default: ''
+    },
+
+    mask: {
+      type: [Array, Function],
+      required: true
```

Comment:
The docs are wrong and need updating.. (I'm working on that)  The mask should be able to accept `false` (boolean).

```js
mask = false
```

This is a new feature (fairly recent), and the docs are out of date